sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs

```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.10.0.0")]
-[assembly: AssemblyFileVersion("0.10.0.0")]
+[assembly: AssemblyVersion("0.11.0.0")]
```

Comment:
Updating assembly info of the test project is not required, as it's not published anywhere.

Reply:
Makes sense.  I've inherited this code and was updating all existing 0.10.0.0 version numbers to 0.11.0.0.  It's OK to leave as 0.11.0.0 right?